M2/Macaulay2/d/version.dd

```diff
@@ -201,21 +198,21 @@ setupconst("version", Expr(toHashTable(Sequence(
 	"),
    "gdbm version" => gdbmversion(),
    "gmp version" => Ccode(constcharstar,"
-	  #ifdef __MPIR_VERSION
+	  #if USING_MPIR
 	       \"not present\"
 	  #else
 	       __gmp_version
 	  #endif
 	  "),
    "mpir version" => Ccode(constcharstar,"
-	#ifdef __MPIR_VERSION
+	#if USING_MPIR
 	     __mpir_version
 	#else
 	     \"not present\"
 	#endif
 	"),
    "ntl version" => Ccode(constcharstar,"NTL_VERSION"),
-   "frobby version" => Ccode(constcharstar,"FROBBY_VERSION"),
+   "frobby version" => "unknown",
```

Comment:
do we not use Frobby anymore? A frobby related chunk further up in this file is also removed.

Reply:
that now reads
```
   "frobby version" => Ccode(constcharstar,"constants::version"),

```

Reply:
Yes, we use frobby, for computing Alexander dual of a matrix.  It is much faster than previous code (and that previous code has, I think, been removed).